sources/packages/web/src/services/AuthService.ts

```diff
@@ -45,7 +45,15 @@ export default async function(
               true,
             );
             isForbiddenUser = true;
-          }
+          } else if (await UserService.shared.checkUser(authHeader) === "Disabled"){
+            await AppConfigService.shared.logout(
+              ClientIdType.INSTITUTION,
+              keycloak,
+              false,
+              true
+            );
+            isForbiddenUser = true;
```

Comment:
Do you need to control if the user is forbidden at this point?

Reply:
![image](https://user-images.githubusercontent.com/77353155/121098009-fac92080-c7c2-11eb-97c9-e5ded42ed1b3.png)
isForbiddenUser = true; was set `true` beacuse of this condition

Reply:
The is fine, but if the user is forbidden, are we not logging off the user and redirecting it to log in before the code reaches this area? Is this "throw new Error" needed?

Reply:
yes, we are logging off and not allowing the user to login.

Reply:
"throw new Error"  was already there. that why I set the flag to true. and the function is the default function. if it doesn't make any sense, we can remove it